packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx

```diff
@@ -3,22 +3,24 @@ import { RelatedComponent } from "vibe-storybook-components";
 import VirtualizedList from "../../../../../components/VirtualizedList/VirtualizedList";
 import { generateItems } from "../../../../../components/VirtualizedList/__stories__/VirtualizedList.stories.helpers";
 import styles from "./virtualized-list.module.scss";
+import { Text } from "src/components";
```

Comment:
same as in the previous PR, the import here is wrong
please run Storybook to make sure your changes work